src/logger/core.ts

```
@@ -6,194 +6,365 @@
-        this.configManager = new LoggerConfigManager();
+  constructor() {
+    this.configManager = new LoggerConfigManager();
+  }
+
+  /**
+     * Built-in output handlers for common use cases
+     */
+  private getBuiltInHandler(type: string, config?: any): LogOutputHandler | null {
+    switch (type) {
+    case 'console':
+      return (level: string, message: string, data?: any) => {
```

ESLint

[error] 'data' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)